targets/unity-v2/source/ExampleTestProject/Assets/Testing/Tests/Client/WebRequestTests.cs

```diff
@@ -0,0 +1,95 @@
+#if !DISABLE_PLAYFABCLIENT_API
+
+using System;
+using System.Collections.Generic;
+using PlayFab.ClientModels;
+using PlayFab.Internal;
+
+namespace PlayFab.UUnit
+{
+    public class WebRequestTests : UUnitTestCase
+    {
+        private Action _tickAction = null;
+        private TestTitleDataLoader.TestTitleData testTitleData;
+        private static readonly PlayFabApiSettings clientSettings = new PlayFabApiSettings();
+        private static readonly PlayFabAuthenticationContext clientContext = new PlayFabAuthenticationContext();
+        
+        private static readonly PlayFabClientInstanceAPI clientInstance = new PlayFabClientInstanceAPI(clientSettings, clientContext);
+
+
+        int originalTimeout = 0;
+        WebRequestType originalWebRequestType = WebRequestType.UnityWebRequest;
+
+        public override void ClassSetUp()
+        {
+            testTitleData = TestTitleDataLoader.LoadTestTitleData();
+            clientSettings.TitleId = testTitleData.titleId;
+        }
+
+        public override void SetUp(UUnitTestContext testContext)
+        {
+            originalTimeout = PlayFabSettings.RequestTimeout;
+            originalWebRequestType = PlayFabSettings.RequestType;
+
+            var titleInfoSet = !string.IsNullOrEmpty(clientSettings.TitleId);
+            if (!titleInfoSet)
+                testContext.Skip(); // We cannot do client tests if the titleId is not given
+        }
+
+        public override void Tick(UUnitTestContext testContext)
+        {
+            if (_tickAction != null)
+                _tickAction();
+        }
+
+        public override void TearDown(UUnitTestContext testContext)
+        {
+            RestoreTimeoutSettings();
+            clientSettings.AdvertisingIdType = null;
+            clientSettings.AdvertisingIdValue = null;
+            _tickAction = null;
+        }
+
+        public override void ClassTearDown()
+        {
+            clientInstance.ForgetAllCredentials();
+        }
+
+[UUnitTest]
+        private void UnityWebRequestTimeOutIgnoredTest(UUnitTestContext testContext)
+        {
+            originalTimeout = PlayFabSettings.RequestTimeout;
+            originalWebRequestType = PlayFabSettings.RequestType;
+
+            PlayFabSettings.RequestTimeout = 1;
+            PlayFabSettings.RequestType = WebRequestType.UnityWebRequest;
+
+            var loginRequest = new LoginWithCustomIDRequest
+            {
+               CustomId = PlayFabSettings.BuildIdentifier,
+               CreateAccount = true,
+            };
+
+            clientInstance.LoginWithCustomID(loginRequest, LoginTimeoutIgnored, LoginTimeoutIgnoredError, testContext);
```

Comment:
I thought that "timed out" was the expected result here?
If we set the timeout to 1ms, and try to call login, it should typically be about 1000x past the timeout by the time it returns.

I think "LoginTimeoutIgnoredError" should be renamed "ExpectLoginTimeoutError"
And maybe "LoginTimeoutIgnored" should be renamed "UnexpectedLoginSuccess"

Reply:
This particular test was ensuring timeout was not adhered to by our WebRequestType.UnityWebReqeust object according to the forum post here: [https://community.playfab.com/questions/37839/request-timeout-ignored-in-unity-sdk.html](https://community.playfab.com/questions/37839/request-timeout-ignored-in-unity-sdk.html) 

---
In reply to: [517627390](https://github.com/PlayFab/SDKGenerator/pull/735#discussion_r517627390) [](ancestors = 517627390)